pkg/apis/v1beta1/ec2nodeclass_status.go

```diff
@@ -63,6 +63,9 @@ type EC2NodeClassStatus struct {
 	// cluster under the AMI selectors.
 	// +optional
 	AMIs []AMI `json:"amis,omitempty"`
+	// PlacementGroups contains the ec2 placement group arns
+	// +optional
+	PlacementGroups []string `json:"placementGroups,omitempty"`
```

Comment:
Should we resolve the strategy into the status block here? If the placement group type is `partition`, should we consider putting the number of partitions that we are using into the status as well?